Source/Core/VideoBackends/OGL/ProgramShaderCache.cpp

```diff
@@ -554,6 +554,13 @@ void ProgramShaderCache::CreateHeader()
 	break;
 	}
 
+	const char* earlyz_string = g_ogl_config.bSupportsEarlyFragmentTests ?
+	            "#extension GL_ARB_shader_image_load_store : enable\n"
```

Comment:
earlyz_string is only used if either ARB_image_load_store or ARB_conservative_depth is supported. 

Only one of the two extensions is enabled.